fedimint-server/src/consensus/mod.rs

```diff
@@ -538,14 +538,20 @@ impl FedimintConsensus {
         client
     }
 
-    pub async fn get_last_epoch(&self) -> Option<u64> {
-        self.db
+    pub async fn get_epoch_count(&self) -> u64 {
+        let maybe_e = self
```

Comment:
`e` is a confusing name here.

Reply:
I've changed it to `ep_hist_key` when using `.map` below.